src/components/login/Registration.js

```diff
@@ -0,0 +1,63 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import "./Login.css";
+
+class Registration extends React.Component{
+  constructor(props){
+    super(props);
+    this.state = {
+      name: "",
+      surname: "",
+      username: "",
+      password: ""
+    };
+
+    this.handleChange = this.handleChange.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  handleChange(event){
+    if(event.target.id === "name"){
+      this.setState({name: event.target.value});
+    }
+    if(event.target.id === "surname"){
+      this.setState({surname: event.target.value});
+    }
+    if(event.target.id === "username"){
+      this.setState({username: event.target.value});
+    }
+    if(event.target.id === "password"){
+      this.setState({password: event.target.value});
+    }
+  }
+
+  handleSubmit(event){
+    event.preventDefault();
+
+    const registration_form = this.state;
```

Comment:
Codacy found an issue: [Identifier 'registration_form' is not in camel case.](https://app.codacy.com/gh/fga-eps-mds/2020.1-VC_Gestor-FrontEnd/pullRequest?prid=6356901)